packages/service/src/utils/documents.ts

```diff
@@ -56,5 +57,6 @@ export function fetchDocument(id: string): Promise<DocumentWithUpdates> {
 export const selectUserDocuments = (user: { id: string }) =>
   db.sql`
 SELECT ${"document"}.* FROM document 
-INNER JOIN ${"user_document"} ON ${"user_document"}.${"document_id"} = ${"document"}.${"id"}
-WHERE ${{ user_id: user.id }}`.run(pool);
+LEFT JOIN ${"user_document"} ON ${"user_document"}.${"document_id"} = ${"document"}.${"id"}
+WHERE ${{user_id: user.id}}
+OR ${{ creator_id: user.id }}`.run(pool);
```

Comment:
So we are keeping user_document? Is this just to have the database's already existing record continuity?

Reply:
there will still need to be a way of sharing the documents with other users. `creator_id` just tracks who created each document.